src/ddmd/func.d

```diff
@@ -156,6 +156,32 @@ enum FUNCFLAGinlineScanned    = 0x20;   /// function has been scanned for inline
 enum FUNCFLAGinferScope       = 0x40;   /// infer 'scope' for parameters
 
 
+/****************************************************
+ * Determine whether an 'out' contract is declared inside
+ * the given function or any of its overrides.
+ * Params:
+ *      fd = the function to search
+ * Returns:
+ *      true    found an 'out' contract
+ *      false   didn't find one
```

Comment:
It's redundant to add the line for `false`, since it must implicitly be the negation of the `true` condition.